WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -526,6 +531,16 @@ - (void)setAvatarForComment:(Comment *)comment forCell:(ReaderCommentCell *)cell
     }];
 }
 
+- (void)checkIfCanComment
+{
+    self.canComment = self.post.commentsOpen && self.isLoggedIn;
+}
+
+- (void)checkIfLoggedIn
+{
+    NSManagedObjectContext *context = [[ContextManager sharedInstance] mainContext];
+    self.isLoggedIn = [[[AccountService alloc] initWithManagedObjectContext:context] defaultWordPressComAccount] != nil;
```

Comment:
(Definitely no need to change anything!). I'm starting to think that we could use a helper, with the default dotcom user always accessible + perhaps all of the accounts. We got this snippet in few other places!